resources/definitions/hctech_hc300-m1&m1h.def.json

```
@@ -1,5 +1,5 @@
 {
-    "version": 2,
+    "version": 3,
```

```suggestion
    "version": 2,
```
This is the version of the file format, not the version how often it was updated